src/services/item/fluent-schema.ts

```diff
@@ -25,7 +25,8 @@ const settings = S.object()
   .prop('isResizable', S.boolean())
   .prop('hasThumbnail', S.boolean())
   .prop('ccLicenseAdaption', S.string())
-  .prop('displayCoEditors', S.boolean());
+  .prop('displayCoEditors', S.boolean())
+  .prop('descriptionPlacement', S.string());
```

Comment:
Not sure but maybe you can have an enum here?

Reply:
Yep, you're right